src/predictor/squats/predictor.py

```diff
@@ -0,0 +1,66 @@
+import cv2
+import tensorflow as tf
+import numpy as np
+import random
+import time
+
+
+class PushupsPredictor:
```

Comment:
1. why you named the class 'PushupsPredictor' in the squats file?
2. Do we need to duplicate 'SquatsPredictor' and 'PushupsPredictor'? because they almost do the same thing ?!